### DISCUSSION TITLE
Poor prediction correlation after QE MD → extxyz conversion: data format issue

### DISCUSSION THREAD

**Discussion by zzzruru** [deepmodeling/deepmd-kit#5009] [Q&A]
Hi Developers, 

I converted the QE md.out file into extxyz format as shown below:

576
energy=-623243.625163 config_type=nep2xyz pbc="T T T" Lattice="19.780973 -0.013912 -0.001887 -0.013912 19.792592 -0.003305 -0.002169 -0.003798 22.621331" Properties=species:S:1:pos:R:3:force:R:3
O 0.034695 0.063708 0.066629 0.415262 0.327405 0.287464
Te 0.000359 0.011593 0.001766 0.111378 -0.552271 -0.230790

Then I used dpdata to split the dataset into training and validation sets.
In the lcurve.out file the training and validation loss curves appear to overlap nicely, yet the correlation between the predicted and original data is very poor.
<img width="960" height="720" alt="lcurve" src="https://github.com/user-attachments/assets/a15c90e2-c3ba-41a5-bf0d-170024d73826" />
<img width="640" height="480" alt="energy" src="https://github.com/user-attachments/assets/639afdc0-0271-439a-97d6-ad32275dfe5b" />

So I would like to ask for help. Could the problem come from the extxyz file format? Is there any recommended way to convert the MD log file output by QE directly with dpdata? Thank you very much for your help!

**Comment by wanghan-iapcm** [comment]
You can verify the dpdata conversion by inspecting a dpdata LabeledSystem instance. For an object `sys`, the atomic coordinates and atom types are accessible via `sys['coords']` and `sys['atom_types']`, respectively.

**Comment by njzjz** [comment]
> O 0.034695 0.063708 0.066629 0.415262 0.327405 0.287464
> Te 0.000359 0.011593 0.001766 0.111378 -0.552271 -0.230790

The coordinates do not seem to be in the unit of Angstrom.
